extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/channels.xml

```diff
@@ -252,4 +252,18 @@
 		<state min="0" max="68399" step="1" readOnly="false">
 		</state>
 	</channel-type>
+
+	<channel-type id="shuffle">
+		<item-type>Switch</item-type>
+		<label>Shuffle</label>
+		<description>Shuffle queue playback</description>
+		<category>Switch</category>
```

Comment:
I would say so - I am anyhow not sure what the "Switch" category is exactly meant to express, but having the channel to be linked to a Switch item type should be good enough as the "switch" icon is then anyhow the default that gets used.